Yafc.UI/ImGui/ScrollArea.cs

```diff
@@ -148,29 +148,51 @@ public float scrollX {
         protected abstract Vector2 MeasureContent(float width, ImGui gui);
 
         public bool KeyDown(SDL.SDL_Keysym key) {
-            switch (key.scancode) {
-                case SDL.SDL_Scancode.SDL_SCANCODE_UP:
-                    scrollY -= 3;
+            bool ctrl = InputSystem.Instance.control;
+            bool shift = InputSystem.Instance.shift;
+            switch ((ctrl, shift, key.scancode)) {
+                case (_, false, SDL.SDL_Scancode.SDL_SCANCODE_UP):
+                    if (ctrl) {
+                        scrollY = 0; // ctrl+up = home
+                    }
+                    else {
+                        scrollY -= 3;
+                    }
                     return true;
```

Comment:
```suggestion
                case (true, false, SDL.SDL_Scancode.SDL_SCANCODE_UP):
                    scrollY = 0; // ctrl+up = home
                    return true;
                case (false, false, SDL.SDL_Scancode.SDL_SCANCODE_UP):
                    scrollY -= 3;
                    return true;
```
I'd like to see the same change for down, left, and right too.